### PR TITLE
Custom Pricing Icon

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/243?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#243** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/243?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#240** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/240?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/components/icons/IconInfoGreenCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      height={size}
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke={color as ColorTokens | undefined}
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      {...otherProps}
+    >
+      <_Circle cx="12" cy="12" r="10" stroke={'#40FB50'} />
```
Comment (0xBigBoss) [inline packages/app/components/icons/IconInfoGreenCircle.tsx:21]: Interesting choice. Wonder if this was accomplish able without this and just using circular attribute with a border.

File: packages/ui/src/components/FormFields/TextField.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ export const TextField = (props: InputProps) => {
         )}
         <Shake shakeKey={error?.errorMessage}>
           {/* @todo: We should make it so that placement can be passed from props */}
+          {/* @todo Write a native component. tooltip doesn't work on native. Maybe wrap into field error */}
```
Comment (0xBigBoss) [inline packages/ui/src/components/FormFields/TextField.tsx:50]: Agreed. Not a big fan of floating error feedback like this.

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/components/icons/IconInfoGreenCircle.tsx (hunk 1 above)
Comment (youngkidwarrior) [reply packages/app/components/icons/IconInfoGreenCircle.tsx:21]: This was originally an icon from `@tamagui/lucide-icons`. Pretty much copy pasted and changed the color. If we have a lot of components like this we can spend time to improve

File: packages/ui/src/components/FormFields/TextField.tsx (hunk 2 above)
Comment (youngkidwarrior) [reply packages/ui/src/components/FormFields/TextField.tsx:50]: I think the designs are leaning toward this style for our errors. Ideally they are persistent until clicked away

but the tamagui tooltip has been a pain to work with. Probably something we should jam with ritesh about

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Mar 28, 11:28 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/243).

**Pull request merged**